keyboards/sofle/keymaps/via/keymap.c

```diff
@@ -17,9 +17,15 @@
 
 #include QMK_KEYBOARD_H
 #include "oled.c"
-#include "encoder.c"
 
-//Default keymap. This can be changed in Via. Use oled.c and encoder.c to change beavior that Via cannot change.
+#if defined ENCODER_ENABLE && defined ENCODER_MAP_ENABLE
```

Comment:
Honestly, only really needs the encoder map enable define check.  It's not added, unless both are enabled (per `builddefs/common_features.mk`, line 940~)
```suggestion
#ifdef ENCODER_MAP_ENABLE
```